makeflow/src/mountfile.h

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright (C) 2008- The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file COPYING for details.
+*/
+
+#ifndef MOUNTFILE_H
+#define MOUNTFILE_H
+
+#include "dag.h"
+
+/* mountfile_parse parses the mountfile and loads the info of each dependency into the dag structure d.
+ * @param mountfile: the path of a mountfile
+ * @param d: a dag structure
+ * @return 0 on success, -1 on failure.
+ */
+int mountfile_parse(const char *mountfile, struct dag *d);
```

Comment:
These functions should be uniformly named as makeflow_mount_* or makeflow_mountfile_* to follow the naming conventions in makeflow.